### DISCUSSION TITLE
Fall League medals not reflecting point threshold changes in profiles

### DISCUSSION THREAD

**Discussion by bcambl** [BattlesnakeOfficial/feedback#133] [Bug Reports]
Recently there was some changes to the fall league point thresholds.
When viewing a developer profile or an individual snake profile, the fall league medal svg icon are still based on the previous point thresholds.

**Comment by bvanvugt** [comment]
Thanks! We'll get this fixed shortly :)

**Comment by chris-bsnake** [comment]
A fix is now deployed
